packages/browser/src/integrations/breadcrumbs.ts

```diff
@@ -313,17 +317,19 @@ function _getFetchBreadcrumbHandler(client: Client): (handlerData: HandlerDataFe
         startTimestamp,
         endTimestamp,
       };
-      const level = getBreadcrumbLogLevelFromHttpStatusCode(data.status_code);
-
-      addBreadcrumb(
-        {
-          category: 'fetch',
-          data,
-          type: 'http',
-          level,
-        },
-        hint,
-      );
+
+      const breadcrumb = {
+        category: 'fetch',
+        data,
+        type: 'http',
+        level: getBreadcrumbLogLevelFromHttpStatusCode(data.status_code),
+      };
+
+      client.emit('outgoingRequestBreadcrumbStart', breadcrumb, {
```

Comment:
m: This naming doe snot make sense I guess - here, the request is ending, not starting. IMHO we can just rename the hook to `beforeOutgoingRequestBreadcrumb` or something like this then?

Reply:
to be clear I mean rename this everywhere like this :)